git/repo.go

```diff
@@ -86,7 +86,7 @@ func (r Repo) Clone(c Config) (*Checkout, error) {
 	}
 
 	notesRef, err := getNotesRef(repoDir, c.NotesRef)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "Couldn't find remote ref") {
```

Comment:
After creating a new blank repo I can't get it to create this error anymore. It must have been something bad in my previous repo. Closing for now.